config/database.travis.yml

```diff
@@ -0,0 +1,8 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+
+test:
+  <<: *default
+  database: travis_ci_test
```

Comment:
Why do we need a database?

Reply:
not for this story but I think at some point we will need a database to keep track of an ongoing game. for example, the database may contain a game table and we'll have a board, players into etc. our endpoint will query the database and perform some logic operations before return the data back to the user.

Reply:
I would remove the database if/until we need it.

Reply:
will remove the database

Reply:
I think there's probably a way to get rid of it completely, but that can be a different PR.